src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  if (logo) {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+  }
+});
```

Comment:
It's a good practice to reject a promise if an error occurs. In this case, if the logo is not found on the page, it would be better to reject the promise with an appropriate error message.